aiger/bv.py

```diff
@@ -481,17 +517,59 @@ def __ge__(self, other):
 
     def rename(self, name):
         """Renames the output of the expression; mostly used internally"""
-        rename_map = {self.name(i): f'{name}[{i}]' for i in range(self.size)}
-        return BV(
-            self.size, (self.variables, self.aig['o', rename_map]), name=name)
+        m = {self.name(i): f'{name}[{i}]' for i in range(self.size)}
+        return BV(self.size, (self.variables, self.aig['o', m]), name=name)
+
+    def __call__(self, args={}, signed=True):
```

Comment:
How hard would it be to add an option to not interpret the result at the end? That is just give the bitvectors? Perhaps a keyword only flag?

`def __call__(self, args=None, signed=True, *, interpret=True):`